lib/oxrdf/Cargo.toml

```diff
@@ -16,13 +16,21 @@ rust-version.workspace = true
 [features]
 default = []
 rdf-star = []
+serde = []
 
 [dependencies]
 oxilangtag.workspace = true
 oxiri.workspace = true
 oxsdatatypes = { workspace = true, optional = true }
 rand.workspace = true
 thiserror.workspace = true
+serde = { version = "1.0.217", features = ["derive"] }
+
+[dev-dependencies]
+# Ideally this would be an optional dependency, used only when the `serde` feature is enabled.
+# However, optional dev-dependencies are not currently supported by cargo.
+# See https://github.com/rust-lang/cargo/issues/1596
+serde_json = { version = "1.0.138" }
```

Comment:
For further info see https://github.com/oxigraph/oxigraph/pull/1172#discussion_r1957361275

Reply:
I'm not sure how to resolve the debian compatibilty error that this still seems to be causing